src/index.cpp

```diff
@@ -1483,6 +1483,10 @@ namespace diskann {
                                                        const unsigned L,
                                                        IdType        *indices,
                                                        float *distances) {
+    if (K > (uint64_t) L)
```

Comment:
I typically advise using curly braces `{...}` even on single-line blocks of code to avoid future bugs if additional lines are added to the block.

Reply:
I understand if there's an established precedent in this project of not putting curly braces on single-line blocks. Possibly leave this to a future refactoring.

Reply:
added {}